src/components/SecurityInsightsWidget/SecurityInsightsWidget.test.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 RoadieHQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import {
+  ApiRegistry,
+  ApiProvider,
+  configApiRef,
+  githubAuthApiRef,
+} from '@backstage/core';
+import { rest } from 'msw';
+import { msw } from '@backstage/test-utils';
+import { MemoryRouter } from 'react-router-dom';
+import { setupServer } from 'msw/node';
+import { alertsResponseMock, entityMock } from '../../mocks/mocks';
+import { SecurityInsightsWidget } from './SecurityInsightsWidget';
+
+const mockGithubAuth = {
+  getAccessToken: async (_: string[]) => 'test-token',
+};
+
+const config = {
+  getOptionalConfigArray: (_: string) => [
+    { getOptionalString: (_: string) => undefined },
+  ],
+};
+
+const apis = ApiRegistry.from([
+  [configApiRef, config],
+  [githubAuthApiRef, mockGithubAuth],
+]);
+
+describe('PullRequestsCard', () => {
+  const worker = setupServer();
+  msw.setupDefaultHandlers(worker);
+
+  beforeEach(() => {
+    worker.use(
+      rest.get(
+        'https://api.github.com/repos/mcalus3/backstage/code-scanning/alerts',
+        (_, res, ctx) => res(ctx.json(alertsResponseMock))
+      )
+    );
+  });
+
+  it('should display an ovreview card with the data from the requests', async () => {

```

Comment:
It'd be nice to test a couple of other cases:
* Where there is an error
* Where there are no issues
* Where the data is loading (https://mswjs.io/docs/api/context/delay - though this doesn't seem like the nicest approach)

I think some of those might be quick enough to add but if any are not we can tackle them in a separate task.

Reply:
I'm a bit stressed about the various errors that appear during setting up these tests. Thought that I'll pave the way and add a first test and mocked responses for all the plugins and then come back (or anyone else at that point) and then add more of them.

Reply:
Sure thing, approved. We can circle back and add more later.